website/docs/r/vra_storage_profile_azure.html.markdown

```diff
@@ -0,0 +1,75 @@
+---
+layout: "vra"
+page_title: "VMware vRealize Automation: vra_storage_profile_azure"
+sidebar_current: "docs-vra-resource-vra-storage-profile_azure"
+description: |-
+  Provides a data lookup for vra_storage_profile_azure.
+---
+
+# Resource: vra_storage_profile_azure
+## Example Usages
+This is an example of how to create a storage profile azure resource.
+
+**Vra storage profile azure:**
+
+```hcl
+# Azure storage profile using generic vra_storage_profile resource. Use 'vra_storage_profile_azure' resource as an alternative.
+resource "vra_storage_profile" "this" {
+  name                = "azure-with-managed-disks"
+  description         = "Azure Storage Profile with managed disks."
+  region_id           = data.vra_region.this.id
+  default_item        = false
+  supports_encryption = false
+
+  disk_properties = {
+    azureDataDiskCaching = "None"         // Supported Values: None, ReadOnly, ReadWrite
+    azureManagedDiskType = "Standard_LRS" // Supported Values: Standard_LRS, StandardSSD_LRS, Premium_LRS
+    azureOsDiskCaching   = "None"         // Supported Values: None, ReadOnly, ReadWrite
+  }
+
+  tags {
+    key   = "foo"
+    value = "bar"
+  }
+}
```

Comment:
Same comment as before to move this to storage profile page and replace it with `vra_storage_profile_azure`.